server/src/LspServer.ts

```diff
@@ -361,24 +356,6 @@ export class LspServer {
     await document?.didSaveTextDocument(true);
   }
 
-  onDidOpenTextDocumentDelayed(params: DidOpenTextDocumentParams): Promise<void> {
```

Comment:
Do we have tests for cases when close request is going right after open?

Reply:
No. And when I tested this behaviour I didn't see close requests right after open... Probably something changed on VS Code side.